ui/page/main_page.go

```diff
@@ -313,10 +357,19 @@ func (mp *MainPage) UnlockWalletForSyncing(wal *dcrlibwallet.Wallet) {
 // displayed.
 // Part of the load.Page interface.
 func (mp *MainPage) HandleUserInteractions() {
+	if mp.IsCurrencySettingUpdated {
```

Comment:
Not sure what is meant by `currency setting`, this name needs some love at least. Also, a global variable of this form to track when a particular setting is changed doesn't sound good. Will propose something later.